src/pages/profile/ProfilePage.tsx

```diff
@@ -0,0 +1,40 @@
+import { User } from 'entities/session/model/type';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Box, Button, Container, Stack, Typography } from 'shared/ui';
+
+type Props = {
```

Comment:
types should be in a separate file in the model

model ----> store.ts | types.ts | index.ts
ui ---------> conponent.tsx | index.ts
index.ts